Nos/Views/FullscreenProgressView.swift

```diff
@@ -6,18 +6,23 @@
 //
 
 import SwiftUI
+import SwiftUINavigation
 
 struct FullscreenProgressView: View {
     
     @Binding var isPresented: Bool 
-    
+
+    var text: TextState?
```

Comment:
`TextState` is a type from `SwiftUINavigation` that adds some features to SwiftUI's default `Text` that we don't need here. In addition the magical extensions we have on `Text` to change the font aren't working for `TextState`. Can you change this to a `String` or `Localized`?
```suggestion
    var text: String?
```

Reply:
[3eaa652](https://github.com/planetary-social/nos/pull/492/commits/3eaa65270aced8302d8a280b4493971fab99b1da)